docs/code-snippets/excel-snippets.yaml

```
@@ -361,397 +229,249 @@ Excel.ChartCollection#add:member(1):
+        
+        console.log("Chart Major Gridlines Format Cleared");
     });
 Excel.ChartFont:class:
   - |-
-    // Set chart title to be Calbri, size 10, bold and in red. 
-    Excel.run(function (ctx) { 
-        var title = ctx.workbook.worksheets.getItem("Sheet1").charts.getItem("Chart1").title;
+    // Set chart title to be Calbri, size 10, bold and in red.
+    await Excel.run(async (context) => { 
+        var title = context.workbook.worksheets.getItem("Sheet1").charts.getItem("Chart1").title;
         title.format.font.name = "Calibri";
         title.format.font.size = 12;
         title.format.font.color = "#FF0000";
         title.format.font.italic =  false;
         title.format.font.bold = true;
         title.format.font.underline = "None";
-        return ctx.sync();
-    }).catch(function(error) {
-        console.log("Error: " + error);
-        if (error instanceof OfficeExtension.Error) {
-            console.log("Debug info: " + JSON.stringify(error.debugInfo));
-        }
+        await context.sync();
     });
 Excel.ChartGridlines#load:member(2):
   - |-
-    // Set to show major gridlines on valueAxis of Chart1
-    Excel.run(function (ctx) { 
-        var chart = ctx.workbook.worksheets.getItem("Sheet1").charts.getItem("Chart1");    
+    // Set to show major gridlines on valueAxis of Chart1.
+    await Excel.run(async (context) => { 
+        var chart = context.workbook.worksheets.getItem("Sheet1").charts.getItem("Chart1");    
         chart.axes.valueAxis.majorGridlines.visible = true;
-        return ctx.sync().then(function() {
-                console.log("Axis Gridlines Added ");
-        });
-    }).catch(function(error) {
-        console.log("Error: " + error);
-        if (error instanceof OfficeExtension.Error) {
-            console.log("Debug info: " + JSON.stringify(error.debugInfo));
-        }
+        await context.sync();
+        
+        console.log("Axis Gridlines Added ");
     });
 Excel.ChartLegend#load:member(2):
   - |-
-    // Get the position of Chart Legend from Chart1
-    Excel.run(function (ctx) { 
-        var chart = ctx.workbook.worksheets.getItem("Sheet1").charts.getItem("Chart1");    
+    // Get the position of Chart Legend from Chart1.
+    await Excel.run(async (context) => { 
+        var chart = context.workbook.worksheets.getItem("Sheet1").charts.getItem("Chart1");    
         var legend = chart.legend;
         legend.load('position');
-        return ctx.sync().then(function() {
-                console.log(legend.position);
-        });
-    }).catch(function(error) {
-        console.log("Error: " + error);
-        if (error instanceof OfficeExtension.Error) {
-            console.log("Debug info: " + JSON.stringify(error.debugInfo));
-        }
+        await context.sync();
+
+        console.log(legend.position);
     });
 Excel.ChartLineFormat#clear:member(1):
   - |-
     // Set to show legend of Chart1 and make it on top of the chart.
```

I don't think this comment describes this sample. 
```suggestion
    // Clear the format of the major gridlines on Chart1. 
```